src/unreadCards/TopicCard.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import React, { PureComponent } from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import TopicCardHeader from './TopicCardHeader';
+import ExtraUnreadCount from './ExtraUnreadCount';
+import DummyMessage from './DummyMessage';
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#FFFFFF',
```

Comment:
Have a look at `themes.js` and see how we reuse styles to make sure components support both `default` and `night` themes.